R/build-articles.R

```diff
@@ -136,16 +141,30 @@ build_rmarkdown_format <- function(pkg = ".",
     render_page(pkg, "vignette", data, path, depth = depth)
   )
 
-  list(
-    path = path,
-    format = rmarkdown::html_document(
+  format <- rmarkdown::html_document(
       toc = toc,
       toc_depth = 2,
       self_contained = FALSE,
       theme = NULL,
-      template = path
-    )
-  )
+      template = path)
+
+  format$knitr$knit_hooks <-
+    list(
+      output = colourise_chunk("output"),
+      message = colourise_chunk("message"),
+      warning = colourise_chunk("warning"),
+      error = colourise_chunk("error"))
+
+  list(path = path, format = format)
+}
+
+colourise_chunk <- function(type) {
```

Comment:
Thank you for spelling colourise correctly

Reply:
I had to look it up, but I knew you would appreciate it :)